packages/frontend/src/components/UserSettingsModal/index.tsx

```diff
@@ -171,6 +172,23 @@ const UserSettingsModal: FC<Props> = ({
                                 }
                             />
                         )}
+                    {orgData &&
+                        !orgData.needsProject &&
+                        user.data?.ability?.can('manage', 'Project') && (
```

Comment:
- Is this permission check correct ? We have no permission check in the BE so I assume the personal tokens are available to anyone. 
- It shouldn't depend on the needsProject either

Reply:
You're right, this was a bad copy/paste job on my part, thanks!!